namer/consul/src/main/scala/io/buoyant/namer/consul/LookupCache.scala

```diff
@@ -1,13 +1,18 @@
 package io.buoyant.namer.consul
 
-import com.twitter.finagle.Address.Inet
+import com.twitter.conversions.time._
 import com.twitter.finagle._
+import com.twitter.finagle.service.Backoff
 import com.twitter.finagle.stats.{NullStatsReceiver, StatsReceiver}
 import com.twitter.util._
 import io.buoyant.consul.v1
-import io.buoyant.namer.Metadata
+
```

Comment:
Linkerd style is to group all imports together (no blank lines) in alphabetical order